src/components/ec-input-field/ec-input-field.vue

```diff
@@ -205,11 +221,19 @@ $ec-input-field-invalid-color: $color-error !default;
     flex-wrap: wrap;
   }
 
+  &__tooltip {
+    align-self: center;
+  }
+
   &__label-text {
     @include input-label;
 
     flex-grow: 1;
     margin-right: 8px;
+
+    &--tooltip {
+      flex-grow: 0;
```

Comment:
I think that enforcing this might cause an issue when the input has label-text, note and label-tooltip. Can you please create that combination in the story and test it?

The `flex-grow: 1` is there to push the note to the right, or push it to the new line when label and note are to long.

Reply:
![image](https://user-images.githubusercontent.com/15978652/84003420-ebba5300-a961-11ea-8322-5b0d521415bc.png)

![image](https://user-images.githubusercontent.com/15978652/84003486-05f43100-a962-11ea-9f8b-d789339e741c.png)

![image](https://user-images.githubusercontent.com/15978652/84003678-523f7100-a962-11ea-9455-8f2fd47c0498.png)

This are I think all the combinations possibly... not sure if it ever happen to have a note with a tooltip?

Reply:
It usually contains validation instructions, so it could be possible. What would happen if we move the tooltip icon inside of `ec-input-field__label-text`?
```
  <span class="ec-input-field__label-text">Short label <ec-icon ... /></span>
```
and keep label-text growing.

Reply:
![image](https://user-images.githubusercontent.com/15978652/84021655-eddddb00-a97c-11ea-8a53-a43ff93995a7.png) 
Was one of my first solutions but then I went mad with the alignment of the icon, I couldn't find a way to align it correctly to the text

Reply:
You can add to the icon:
```
display: inline-block;
vertical-align: baseline? bottom? text-top? // one of those
```

or you can make `ec-input-field__label-text` display: flex or inline-flex and position children inside using flex properties.

Reply:
ohh, ok in the end `ec-input-field__label-text display: flex` works with the icon `align-self: center;`
I'll update the tests and push again - thanks